charts/cloudserver/templates/tests/test-cloudserver-service.yml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: Pod
 metadata:
-  name: {{ template "cloudserver-front.fullname" . }}-test
+  name: {{ template "cloudserver.fullname" . }}-test
```

Comment:
Since you are modifying this value already, a better name will be:
`{{ template "cloudserver.fullname" . }}-service-test`